sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs

```diff
@@ -214,8 +364,72 @@ public async Task ProcessEventsAsync(EventProcessorHostPartition context, IEnume
                 }
             }
 
+            private async Task TriggerExecute(EventData[] events, EventProcessorHostPartition context, CancellationToken cancellationToken)
+            {
+                var triggerInput = new EventHubTriggerInput
+                {
+                    Events = events,
+                    ProcessorPartition = context
+                };
+
+                // Batch dispatch
+                TriggeredFunctionData input = new()
+                {
+                    TriggerValue = triggerInput,
+                    TriggerDetails = triggerInput.GetTriggerDetails(context)
+                };
+
+                await _executor.TryExecuteAsync(input, cancellationToken).ConfigureAwait(false);
+
+                // The timer should reset upon invocation of the function
+                _storedEventsBackgroundTask = null;
```

Comment:
Seems like we should signal a cancellation token and then await the task.  How does this stop the task from doing stuff?